### DISCUSSION TITLE
Scheduled Transactions

### DISCUSSION THREAD

**Discussion by ColGra** [actualbudget/releases#239] [Q&A]
I might be missing a trick but I just cannot see a way to get scheduled transactions to post with a category. I have tried linking transactions but the box does not remain ticked after saving?

Comments

Share

Save

**Comment by triplejranch** [comment]
I just answered this on Reddit.  Go to Edit as a Rule after you have saved the schedule.  Then, under the " Then apply these actions" it will let you add a category or note, etc.

**Comment by ColGra** [comment]
Thank you :-)
